src/views/Signup.jsx

```diff
@@ -0,0 +1,146 @@
+import svg from "../assets/signup_svg.svg";
+import barefootLogo from "../assets/barefoot_logo.svg";
+import { Link, useNavigate } from "react-router-dom";
+import React, { useState } from 'react';
+import axios from 'axios';
+import swal from 'sweetalert2';
+import signupValidation from '../service/validations/signup';
+
+const Signup = () =>{
+  const navigate = useNavigate()
+  const [userRequest, setUserRequest] = useState({buttonDisable:false,email:'',lastName:'',role:'requester',phoneNumber:'',gender:'select',password:'',repeatPassword:'',firstName:'',userName:''});
```

Comment:
this kind of state will be a headache to maintain, group information which are related